sql/core/src/test/scala/org/apache/spark/sql/sources/v2/DataSourceV2SQLSuite.scala

```diff
@@ -344,4 +345,834 @@ class DataSourceV2SQLSuite extends QueryTest with SharedSQLContext with BeforeAn
         df_joined)
     }
   }
+
+  test("AlterTable: table does not exist") {
```

Comment:
can we create a new test suite for alter table? I'm afraid this test suite will get too big if we put all the DDL tests here.